### ISSUE TITLE
Strange issues with Siri, no changes made to Homebridge

### ISSUE THREAD

**Issue by srb37** [nfarina/homebridge#608]
Recently home bridge has become pretty much unusable.  I've tried updating to the latest version of code, but the issues remain.

I'm mainly using the homebridge-http plugin and the Http-Multi plugin to control lights and shades through URLs.

I'm having two major issues:
1) Recently the shades stopped responding to commands via Siri.  I'd say "Open the Bedroom Shades" Siri would understand me, but would respond "Sorry, srb37, I wasn't able to find any bedroom shadeses (sic) this time."  No entries would appear on the home bridge console.

2) I'd ask, "Turn off/on the Kitchen Lights" randomly it would work fine and other times it would trigger all of my lights to go on at once with the following output on the console:
[3rd Floor Stairway] Setting power state to off
[Dining Room] Setting power state to off
[Bedroom] Setting power state to off
[Living Room] Setting power state to off
[Music Room] Setting power state to off
[Kitchen] Setting power state to off
[Great Room TV] Setting power state to off
[Nighttime] Setting power state to off
[Breakfast] Setting power state to off
[3rd Floor] Setting power state to off
[Downstairs] Setting power state to off
[Guest Bedroom] Setting power state to off
[Upstairs] Setting power state to off
[Stairway Light] Setting power state to off
[Dinner Lights] Setting power state to off
[Cooking Lights] Setting power state to off
[Great Room] Setting power state to off
[Nursery] Setting power state to off
[3rd Floor Office] Setting power state to off
[Great Room TV] HTTP power function succeeded!

Any ideas?

**Comment by DaronLord** [comment]
I and have the same issue.  If I try to address any individual device to be turned ON or OFF under every circumstance all HomeKit devices are turned ON, even devices not associated with homebridge.  If I remove homebridge and the associated devices the problem goes away.  However, if I add a device controlled by homebridge to a scene and trigger the scene via Siri or an App it works without issue.  Periodically, if I ask Siri to adjust my Nest thermostats it will say no devices found. If I wait a few seconds and try again it works.

**Comment by spectre3ooo** [comment]
I'm having a similar issue with my garage doors.  If I try to open one of them by name, it can't find anything.  But, if I say "Open all of my garage doors", then they both open.  It's like Siri can find the devices by type, but not by name.  I can tell she recognizes "Left Garage Door" and "Right Garage Door" because she matches the capitalization in my config file...

**Comment by psmith3** [comment]
This happens to me a lot as well. I use the Home app and I think it is an issue with the app. When I first setup the app, I edit the doors and change the service name to match the actual names of the doors. The service name is what Siri uses. By default the service name is "Garage Door". If you have not changed the service name of each door, that is your problem. However, sometimes Siri does not remember the service name. When that happens, I blow away the settings in Home app and also the Persist folder on the server and start over.

**Comment by JerGitHub** [comment]
@spectre3ooo @psmith3 Are you guys using the Liftmaster Homebridge plug-in?

This just started happening to me when I updated my homebridge to v0.3.3 and iOS 9.3.1 using the Liftmaster plug-in. I will try blowing away the settings and persist and see what happens but that is a pain in the ass since I have a bunch of other stuff setup from other plugins and HomeKit devices.

**Comment by spectre3ooo** [comment]
I created a plugin to control and sense my doors using the GPIO pins on the pi, but I based the code on the liftmaster plugin. So, from homebridge's point of view, it's very similar.  

I also have another phone that is running 9.3 (not 9.3.1).  I set up a completely new apple ID and home on it and it does the same thing.

**Comment by JerGitHub** [comment]
Interesting....did you blow away your persist and homekit data and start over? I haven't had a chance to try @psmith3 suggestion to see if that works.

**Comment by srb37** [comment]
I'm glad I'm not the only one with these issues.  A couple of other things about my setup and troubleshooting steps:
When I initially setup home bridge, I used MyTouchHome (remember everything was working fine up until a few weeks ago).

To troubleshoot, here are some of the things I've tried:
1) Updated HomeBridge and all accessories
2) Deleted the persist folder
3) Cleared homekit configuration on my iPhone both in the main HomeKit settings and in Privacy/HomeKit.
4) Changed from the homebridge-http plugin to the homebridge-readablehttp and homebridge-httpmulti plugins.
5) Used the Home app instead of MyTouchHome to setup the system.

All of those steps seemed to yield the same results.  I'll continue to play around, I can't seem to find a way in the app to change the service name.  In the config.json file, each accessory is uniquely named, for the homebridge-http accessory there isn't a service name field in the config file.

**Comment by AppleTechy** [comment]
FYI: If you use Insteon+ it labels the service name and the user names.
When u go into the device details.

**Comment by psmith3** [comment]
Yeas, I am using the same Liftmaster plug-in.

**Comment by spectre3ooo** [comment]
@JerGitHub Yeah, I deleted the persist folder.

**Comment by JerGitHub** [comment]
I use the latest Eve app and they call it "Accessory Name" which you need to modify so it's not the same name as the Room.

**Comment by kalel2000** [comment]
This happened to me too just one day out of the blue. I even tried rebuilding my homebridge, but that didn't help.  Then he time when trying to turn on a light instead of saying "Desk Light" I said just "Desk". And it worked fine.  In wink I have the light labeled as "Desk" not "Desk Light". I think that there was a change in o Siri that caused this.  I used the Insteon+ app the change the Siri name of all my lights to be what I am used to calling them and haven't had a problem since. That was two months ago.  Hope this helps.

**Comment by desvdp** [comment]
Got the same issue, kind off...
to me it feels like it's accessory type dependent. I'm currently experimenting with a simple setup, just 2 types of devices (Hue lamps and the denon-avr, which is implemented a switch accessory). When I ask Siri to "Turn on all lights" it will always behave as expected, however when I ask Siri to turn on the AVR it happens all the lights will go on along with it (most of the time actually)

**Comment by srb37** [comment]
I think I figured out what the issue was.  All of my devices were named by what they controlled so "Bedroom Lights" would turn off the lights in the bedroom.  I therefore didn't bother to create rooms and such, leaving all devices in their default location.  I finally downloaded the Insteon+ app and created rooms to put each device in.  So I'll have a room called "Bedroom" and in that room I have two devices the "Bedroom Lights" and the "Bedroom Shades".  Now when I say "Turn off the Bedroom lights", Siri responds appropriately, likewise with the shades.
